fastlane.gemspec

```diff
@@ -54,7 +54,12 @@ Gem::Specification.new do |spec|
   spec.homepage      = "https://fastlane.tools"
   spec.license       = "MIT"
   spec.metadata      = {
-    "docs_url" => "https://docs.fastlane.tools"
+    "bug_tracker_uri" => "https://github.com/fastlane/fastlane/issues",
+    "changelog_uri" => "https://github.com/fastlane/fastlane/releases",
+    "documentation_uri" => "https://docs.fastlane.tools/",
+    "homepage_uri" => spec.homepage,
+    "mailing_list_uri" => "https://github.com/fastlane/fastlane/discussions",
```

Comment:
I'd love to have a link to discussions but mailing list seems a little off 🤷‍♂️ I think we are good to ship after removing this one!
```suggestion
```